app/components/DropdownMenu/DropdownMenu.js

```diff
@@ -51,8 +53,9 @@ class DropdownMenu extends React.Component<Props> {
   ) => {
     return (ev: SyntheticMouseEvent<HTMLElement>) => {
       ev.preventDefault();
+      const { t } = this.props;
       const currentTarget = ev.currentTarget;
-      invariant(document.body, "why you not here");
+      invariant(document.body, t("why you not here"));
```

Comment:
This is a development warning, no need to translate

Reply:
removed